CONTRIBUTING.md

```diff
@@ -64,6 +65,32 @@ issue yourself. This gives the team visibility of the work you plan to do.
 The other advantage to raising the issue at the outset is that our process requires an
 issue to be associated with every PR (see [patch format](#patch-format)).
 
+### Before submitting a PR
+
+Kata Containers utilizes a number of [CI systems](https://en.wikipedia.org/wiki/Continuous_integration)
+to automatically check every PR. As well as performing execution tests locally on
+your PR before submission, you are also strongly encouraged to run the suite of
```

Comment:
Removing "also": your PR before submission, you are strongly encouraged to run the suite of